lib/apipie_bindings/api.rb

```diff
@@ -213,7 +213,7 @@ def http_call(http_method, path, params={}, headers={}, options={})
       if dry_run?
         empty_response = ApipieBindings::Example.new('', '', '', 200, '')
         ex = options[:fake_response ] || empty_response
-        response = create_fake_response(ex.status, ex.response, http_method, path, args)
+        response = create_fake_response(ex.status, ex.response, http_method, URI.join(@uri, path).to_s, args)
```

Comment:
Thanks, I reproduced the test failures and have updated it to use example.com when in dry_run mode and the URI is missing.  I've reverted other changes to the test so it no longer passes a URI.